instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/version.py

```diff
@@ -13,5 +13,3 @@
 # limitations under the License.
 
 __version__ = "0.51b0.dev"
-
-_instruments = tuple()
```

Comment:
What's wrong with this?

Reply:
Oh, it's not used. There's a `_instruments` on the `package.py`. This is the `version.py`.